resources/views/components/chirps/chirp-card.blade.php

```diff
@@ -0,0 +1,143 @@
+                    {{ $comment->comment }}
+                    @if(auth()->user() && auth()->user()->id === $comment->user_id)
+                        <form method="POST" action="{{ route('comments.destroy', $comment) }}" data-id="{{ $comment->id }}"class="delete-comment lg:flex lg:items-center">
+                            @csrf
+                            <input type="hidden" name="comment_id" value="{{ $comment->id }}">
+                            <input type="hidden" id="user_id" name="user_id" value="{{ Auth::user() }}">
+                            <button type="submit" class="deleteButton bg-red-500 text-white py-1 px-2 m-1 rounded hover:bg-red-800">Delete</button>
+                        </form>
+                    @endif 
+                </div>
+            @endforeach
+        </div>
+        <form method="POST" action="{{ route('comments.store') }}" class="post-comment mt-4 lg:flex lg:items-center" name="commentForm">
+            @csrf
+            <input type="hidden" name="chirp_id" value="{{ $chirp->id }}">
+            <textarea name="comment" maxlength="255" class="w-full lg:w-1/4 p-1 border border-blue-500 rounded" placeholder="Add a comment" required></textarea>
+            <button type="submit" class="mt-2 bg-blue-500 text-white py-2 px-4 rounded ml-2 hover:bg-blue-800">Comment</button>
+        </form>     
+    </div>
+</div>
+
+<script>
+    function deleteComments(){
+        let forms = document.querySelectorAll("form.delete-comment");
+        forms.forEach(deleteButtonForm => {
+            deleteButtonForm.addEventListener("submit", async (event) => {
+                event.preventDefault();
+                const url = deleteButtonForm.action
+                const formData = new FormData(deleteButtonForm)
+
+                console.log(url)
+                let response = await fetch(url, {
+                    method: "POST",
+                    body: formData
+                });
+                console.log(await response.json());
+                deleteButtonForm.parentElement.remove();
+            })
+        })
+    }
+    function renderComments(){
+        let forms = document.querySelectorAll('form.post-comment');
+        forms.forEach(commentButtonForm => {
+            commentButtonForm.addEventListener('submit', async (event) => {
+                event.preventDefault();
+                let url = commentButtonForm.action;
+                const formData = new FormData(commentButtonForm);
+                console.log(url);
+
+                let response = await fetch(url, {
+                    method: "POST",
+                    body: formData,
+                })
+                .then(response => {
+                    return response.json();
+                })
+                .then((data) => {
+                    let newest = document.querySelector(".comment")
+                    if (!newest) {
+                        newest = document.createElement("div")
```

Comment:
when I add a comment with non on the page the comment does not have the border or styles like the other one

Reply:
Will be fixed in next commit!